source/speech.py

```diff
@@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
 			res.append(symbol)
 		else:
 			res.append(u"{count} {symbol}".format(count=count, symbol=symbol))
-
-	return " ".join(res)
+		quarterTones += (count*4 if raw[0]== "\t" else count)

```

Comment:
My apologies. You're quite correct. That's what I get for code reviewing late at night. :)